packages/features/sales-escrows/modals/StartAuctionModal.tsx

```diff
@@ -45,7 +45,21 @@ const validationSchema = Yup.object({
     .default(dateNow),
 });
 
-export function StartAuctionModal({ currentToken, open, handleClose, onSuccess }: any) {
+interface StartAuctionModalProps {
+  currentToken: string;
+  open: boolean;
+  handleClose: (any) => void;
+  onSuccess: (any) => Promise<void>;
+  setInProcess: (any) => void;
```

Comment:
1. Any should be boolean
2. Please rename to onProcessing
3. Please make it optional